guides/relationships/features/links-mode.md

```diff
@@ -0,0 +1,199 @@
+
+```ts
+interface FetchRelationshipRequest {
+  op: 'findHasMany' | 'findBelongsTo';
+  store: Store;
+  url: string; // the related link
+  method: 'GET';
+  records: StableRecordIdentifier | StableRecordIdentifier[] | null; // the current membership of the relationship
+  data: {
+    useLink: true;
+    field: LegacyBelongsToField | LegacyHasManyField;
+    links: Links;
+    meta: Meta;
+    options: unknown; // any options passed to `reload` or `load`
+    record: StableRecordIdentifier; // the parent record
+  };
+
+  // tells the store not to automatically convert the response into something reactive
+  // since the reactive relationship class itself will do that
+  [EnableHydration]: false; 
+}
+```
+
+The most important 3 things in this request are
+
+- the `op` code: this is how the cache will know to use the response to update the state of a relationship
+- `data.field`: this is how the cache will know which field it should update
+- `data.record`: this is how the cache will know which record to associate the response to.
+
+The normalized API response (what your handler must return either directly from your API or with some normalization on the client) that should be passed to the JSON:API cache should be a standard JSON:API document.
+
+The contents of `data` will be inserted into the resource cache and the list of records contained used to update the state of the relationship. The `meta` and `links` of the response will become the `meta` and `links` available for the
+relationship as well.
+
+Sideloads (included records) are valid to include in these response.
+
+## Activating LinksMode
+
+### For A Relationship on a Model
+
+Add `linksMode: true` to the relationship's options.
+
+```ts
+import Model, { belongsTo, hasMany } from '@ember-data/model';
+
+export default class User extends Model {
+  @belongsTo('address', {
+    async: false,
+    inverse: 'residents',
+    linksMode: true
+  })
+  homeAddress;
+}
+```
+
+This works for both `async` and `non-async` relationships and only changes the fetching behavior of the field it is defined on. So for instance, in the example above, `homeAddress` is fetched in links mode while `<Address>.residents` might still be using the legacy adapter experience.
+
+### For A SchemaRecord in LegacyMode
+
+```ts
+import type { ResourceSchema } from '@warp-drive/core-types/schema/fields';
+
+const UserSchema = {
+  type: 'user',
+  // this is what puts the record instance into legacy mode
+  legacy: true,
+  fields: [
+    {
+      kind: 'belongsTo',
+      name: 'homeAddress',
+      options: {
+        async: false,
+        inverse: 'residents',
+        linksMode: true
+      }
+    }
+  ]
+} satisfies ResourceSchema;
+```
+
+The behavior of relationships for a SchemaRecord in LegacyMode is always identical to that of Model's.
```

Comment:
```suggestion
The behavior of relationships for a SchemaRecord in LegacyMode is always identical to that of Models.
```

Reply:
`Model's` is correct because it is a shortened possessive, though you could approach it from the perspective of model instances too...